src/lib/stepper/stepper.md

```diff
@@ -112,10 +112,52 @@ are completed.
 If completion of a step in linear stepper is not required, then the `optional` attribute can be set
 on `mat-step`.
 
+```html
```

Comment:
Actually you can remove these and put magic comments like for the overview example above:

```html
<!-- example(stepper-overview) -->
```

Just replace `stepper-overview` with the name of the example folder and we have a script to embed it when we suck the markdown files into the docs site. (It comes out looking like this: https://material.angular.io/components/stepper/overview)